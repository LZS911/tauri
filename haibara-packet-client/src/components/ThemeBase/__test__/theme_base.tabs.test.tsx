import { render } from '@testing-library/react';
import { TabPane, Tabs } from '../Tabs';

describe('test Tabs', () => {
  test('should match snapshot', () => {
    const { container } = render(
      <Tabs>
        <TabPane>children</TabPane>
      </Tabs>
    );

    expect(container).toMatchSnapshot();
  });
});
