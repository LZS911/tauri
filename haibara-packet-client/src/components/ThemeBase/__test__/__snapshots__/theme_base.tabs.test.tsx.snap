// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Tabs should match snapshot 1`] = `
<div>
  <div
    class="ant-tabs ant-tabs-top dark:text-white "
  >
    <div
      class="ant-tabs-nav"
      role="tablist"
    >
      <div
        class="ant-tabs-nav-wrap"
      >
        <div
          class="ant-tabs-nav-list"
          style="transform: translate(0px, 0px);"
        >
          <div
            class="ant-tabs-tab ant-tabs-tab-active"
          >
            <div
              aria-controls="rc-tabs-test-panel-null"
              aria-selected="true"
              class="ant-tabs-tab-btn"
              id="rc-tabs-test-tab-null"
              role="tab"
              tabindex="0"
            />
          </div>
          <div
            class="ant-tabs-ink-bar ant-tabs-ink-bar-animated"
          />
        </div>
      </div>
      <div
        class="ant-tabs-nav-operations ant-tabs-nav-operations-hidden"
      >
        <button
          aria-controls="rc-tabs-test-more-popup"
          aria-expanded="false"
          aria-haspopup="listbox"
          aria-hidden="true"
          class="ant-tabs-nav-more"
          id="rc-tabs-test-more"
          style="visibility: hidden; order: 1;"
          tabindex="-1"
          type="button"
        >
          <span
            aria-label="ellipsis"
            class="anticon anticon-ellipsis"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="ellipsis"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M176 511a56 56 0 10112 0 56 56 0 10-112 0zm280 0a56 56 0 10112 0 56 56 0 10-112 0zm280 0a56 56 0 10112 0 56 56 0 10-112 0z"
              />
            </svg>
          </span>
        </button>
      </div>
    </div>
    <div
      class="ant-tabs-content-holder"
    >
      <div
        class="ant-tabs-content ant-tabs-content-top"
      >
        <div
          aria-hidden="false"
          aria-labelledby="rc-tabs-test-tab-null"
          class="ant-tabs-tabpane ant-tabs-tabpane-active"
          id="rc-tabs-test-panel-null"
          role="tabpanel"
          tabindex="0"
        >
          children
        </div>
      </div>
    </div>
  </div>
</div>
`;
